cypher/cypher-docs/src/docs/dev/query-tuning-indexes.adoc

```diff
@@ -15,10 +15,83 @@ Therefore, node and relationship indexes are used interchangeably in this sectio
 
 Please refer to <<administration-indexes-search-performance>> for instructions on how to create and maintain the indexes themselves.
 
+== Index types and predicate compatibility
 
-include::ql/administration/indexes/node-index-example.asciidoc[leveloffset=2]
+There are different types of indexes available in Neo4j.
+Not all indexes are compatible with the same property predicates.
 
-include::ql/administration/indexes/relationship-index-example.asciidoc[leveloffset=2]
+In the most common case, indexes are used for `MATCH` and `OPTIONAL MATCH` clauses that combine a label predicate with a property predicate.
+Therefore, it is important to know the kind of predicates that can be solved by the different indexes.
+
+=== BTREE indexes
+
+`BTREE` indexes support all types of predicates:
```

Comment:
Should we mention that BTREE allows composite indexes while text doesn't?

Reply:
I have been thinking about it, too.
It is a question if this is the place. We don't allow creating a composite text index, so a user will definitely not be wondering "why did the planner not pick up my composite text index?"